catalyst-generator/src/main/resources/templates/usecase/mqtt/DemoApplication.java

```diff
@@ -0,0 +1,17 @@
+{{=<% %>=}}
```

Comment:
We have the **Application.java** as a template for Spring Boot app - why do we need a DemoApplication, which has no specific configurations?